Subject: [PATCH] Bazel support

---
 common/platform.cpp | 2 +-
 core/common.h       | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git common/platform.cpp common/platform.cpp
index 120a8ca..657a43d 100644
--- common/platform.cpp
+++ common/platform.cpp
@@ -4,7 +4,7 @@
 #include "platform.h"
 
 #if defined(OIDN_X64)
-  #include "mkl-dnn/src/cpu/x64/xbyak/xbyak_util.h"
+  #include "src/cpu/x64/xbyak/xbyak_util.h"
 #endif
 
 namespace oidn {
diff --gitcore/common.hcore/common.h
index 7906013..51220d8 100644
--- core/common.h
+++ core/common.h
@@ -12,7 +12,7 @@
 #include "vec.h"
 
 #if defined(OIDN_DNNL)
-  #include "mkl-dnn/include/dnnl.hpp"
+  #include "include/dnnl.hpp"
 #elif defined(OIDN_BNNS)
   #include <Accelerate/Accelerate.h>
 #endif
-- 
2.39.1

